src/main.py

```diff
@@ -1,6 +1,17 @@
-def main():
-    pass
+def main ():
+    for number in range (0, 100):
+        number +=1
```

Comment:
rangeでこのように指定することもできます

```suggestion
    for number in range (1, 101):
```

Reply:
さらに、最大数を定数として切り出すと、より一層見通しがよくなります。

```suggestion
    MAX_NUMBER = 100
    for number in range (0, MAX_NUMBER + 1):
```

「1から100まで」から「1から500まで」に変えて言われたときに、 `MAX_NUMBER` の値を変更するだけでよくなります。

Reply:
あ〜そうですね、確かに。
ありがとうございます！

Reply:
ここではコメントするのが難しかったからやってないけど、実際にはmain()の外に `MAX_NUMBER` を書くのがいいね

Reply:
MAX_NUMBERですか。確かに。
そっちの方が見やすくもありますね、レビューしやすいと言いますか。